### PR TITLE
List and rating

### PR DESCRIPTION
**Pull request by purvasharmadev**
# Changes made:

- Added List component > Simple ordered list, simple unordered list, simple stacked list.
- Added Classes for Rating system.

## files to check:

- list.html, list.css
- rating.html, rating.css

### CODE REVIEW

**Comment by Ranjeetk489** [comment]
Preview is Missing

**Review by tonymike99 — APPROVED** [review]
Hi Purva,
The code is neatly aligned and structured.
It is also very easy to read and understand.
Suggestion: You could have added the deploy link to visually see the changes.
No problems with the code. 
Good to merge. 👍🏻

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by Shubhamkr2610 — APPROVED** [review]
code is clean and readable , Easy to understand . Everything looks good to me . You can merge it

File: Components/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.star-rating {
+  box-shadow: 0px 0px 5px orange;
+}
+
+.star-rating-checked {
+  color: orange;
```
Comment (sumitk-singh) [inline Components/Rating/rating.css:6]: suggestion: use color variable.

**Review by sumitk-singh — APPROVED** [review]
Have a look at a little suggestion that i have mentioned.
although code looks clean and readable. Good to go.

**Pull request merged**